aws-s3-transfer-manager/tests/e2e_transfer_test.rs

```diff
@@ -305,3 +304,89 @@ async fn test_upload_with_long_running_stream() {
         }
     }
 }
+
+#[tokio::test]
+async fn test_empty_object_download() {
+    let (tm, _) = test_tm().await;
+    let (bucket_name, _) = get_bucket_names();
+
+    let object_key = "pre-existing-empty";
+
+    let mut download_handle = tm
+        .download()
+        .bucket(bucket_name)
+        .key(object_key)
+        .initiate()
+        .unwrap();
+
+    let body = drain(&mut download_handle).await.unwrap();
+
+    assert_eq!(body.len(), 0);
+}
+
+async fn range_download_helper(
```

Comment:
This helper is only 3 lines and used at one place. Should we just get rid of it for now?